.github/generate_ci.php

```diff
@@ -0,0 +1,290 @@
+<?php
+
+if (!function_exists('yaml_emit')) {
+    fwrite(STDERR, "yaml extension is required\n");
+    exit(1);
+}
+
+const GENERATION_WARNING = <<<COMMENT
+#############################################################
+#   WARNING: automatically generated file, DO NOT CHANGE!   #
+#############################################################
+
+# This file was automatically generated. Adjust and run the
+# .github/generate_ci.php file to make changes to the
+# pipeline.
+
+
+COMMENT;
+
+function generate() {
+    generate_push();
+    generate_nightly();
+}
+
+function generate_push() {
+    $result = [
+        'name' => 'Push',
+        'on' => ['push' => ['ignore-paths' => [
+            'docs/*',
+            'NEWS',
+            'UPGRADING',
+            'UPGRADING.INTERNALS',
+            'README.md',
+            'CONTRIBUTING.md',
+            'CODING_STANDARDS.md',
+        ]]],
+        'jobs' => [
+            'DEBUG_NTS' => linux_x64(
+                configuration_parameters: '--enable-debug --disable-zts',
+                extended_tests: false,
+            ),
+            'RELEASE_ZTS' => linux_x64(
+                configuration_parameters: '--disable-debug --enable-zts',
+                extended_tests: false,
+            ),
+            // apt broken, not sure why
```

Comment:
Broken how? Apt on i386 is always a bit fragile. Might make sense to switch it to use docker.

Reply:
https://github.com/iluuu1994/php-src/runs/5145501102?check_suite_focus=true